current/local-cli/rnpm/windows/index.js

```diff
@@ -15,6 +15,10 @@ module.exports = [{
   }, {
     command: '--template [template]',
     description: 'Template to install. E.g., `vnext`.',
+  }, {
+    command: '--language [language]',
+    description: 'Which language the template app is written in.  Possible values are cs and cpp',
```

Comment:
> Possible [](start = 65, length = 9)

Remove possible

Reply:
I prefer this language.

---
In reply to: [329279587](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r329279587) [](ancestors = 329279587)